### ISSUE TITLE
Apoiar uma oportunidade

### ISSUE THREAD

**Issue by augustohp** [iMastersDev/oportunidades#21]
É importante diferenciar as oportunidades bacanas das ruins. É sacanagem identificar as _ruins_ de maneira explícita, até porque o critério pra isso é muito pessoal.

Então vamos dar a oportunidade de quem estiver cadastrado ou identificado ou quaquer coisa de dar uma espécie de _like_ em uma vaga. Indicando a vaga como bacana.

O que acham?

**Comment by netojoaobatista** [comment]
+1 para a feature.

Seria o mesmo que o sugerido para empresas na #16, só que dessa vez, para a oportunidade em si. De fato, se o +1 for somente para a oportunidade, podemos classificar "boas empresas" baseado no número de boas oportunidades.

**Comment by augustohp** [comment]
Acho melhor dessa forma. Porque nada garante que a empresa sempre postará boas vagas ou sempre estará na mesma "_boa_" situação...

**Comment by netojoaobatista** [comment]
Concordo 100% @augustohp, com o +1 para a oportunidade, teremos uma métrica mais real para a empresa. 

Podemos até tirar o status de `feature` da Issue #16.

**Comment by iannsp** [comment]
realmente a atuação em qualificacao de maneira positiva ao inves de negativa permite
a) menos problemas com as empresas por estarem "negativadas" ja que vamos estar gerando analise positiva.
b) esse é um comportamento mais "social"

vejo isso como uma coisa interessante, um exciter mesmo, e o colocaria no backlog. Não rola a implementacao dessa issue agora.

**Comment by alganet** [comment]
Eu gosto dessa feature, é simples de programar e resolve uma pica gigantesca: moderação. Ninguém quer ficar o dia todo moderando vagas. Se o usuário puder ver quais vagas são melhores de alguma forma, ele pode aprender a ignorar as que não tem likes.

**Comment by augustohp** [comment]
Como vocês mesmo disseram: ela resolve uma pica gigantesca. Várias picas na verdade. O _Hacker News_ funciona excepcionalmente bem só com isso.

**Comment by diegoholiveira** [comment]
+1

O que vocês acham de criar uma feature que, baseada na aceitação da vaga, o sistema vá sugerindo "revisões" para vaga?

Algo do tipo: "Oi, notamos que seu anúncio não é popular, sugerimos que você faça uma revisão no seu anúncio."

Um criterio que sugiro para ser a aceitação da vaga é o numero de likes + numero de visualizações.

**Comment by netojoaobatista** [comment]
@diegoholiveira, penso que isso é problema da empresa, ela que deve monitorar suas oportunidades.

Podemos, no futuro, listar as vagas por dois critérios:
1. Preferência do usuários (skills que ele pretende ver).
2. Popularidade da oportunidade.
3. Data da oportunidade.

Obviamente, a questão dos skills que o usuário pretende visualizar é uma `feature` futura, não será incluída nesse milestone, mas quando tivermos um perfil de usuário, podemos permitir que ele selecione algumas características das oportunidades que ele pretende ver, como localização, skills, faixa salarial, etc.

Se a empresa perceber que sua _oportunidade_ não está sendo visualizada, fica a cargo dela corrigir isso. Não precisamos notificá-la.
